images/ubuntu/19.04/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM docker.io/library/ubuntu:19.04
+
+ENV NAME=ubuntu-toolbox VERSION=19.04
+LABEL com.github.debarshiray.toolbox="true" \
```

Comment:
We should add `com.github.containers.toolbox="true"` to the list. See commit de5e5df9b793edc746cde7125cd437b69f15ccd7.